rcon/extended_commands.py

```diff
@@ -968,7 +969,7 @@ def get_map_rotation(self):
         return l
 
     def do_add_map_to_rotation(
-        self, map_name, after_map_name: str = None, after_map_name_number: str = None
+        self, map_name, after_map_name: str, after_map_name_number: str = None
```

Comment:
You really should keep `after_map_name` optional. When not provided, you may assume that the user intends to append the map to the end of the rotation. That way we preserve the function's signature and behavior.

Reply:
Then we would need to query the current map rotation before, isn't it? As the second argument is now not optional anymore.

Reply:
Yes.